http_outlet_test.go

```diff
@@ -121,9 +199,9 @@ func TestTimeout(t *testing.T) {
 	config.SkipVerify = true
 	config.Timeout = 100 * time.Millisecond
 
+	logCapture := new(bytes.Buffer)
```

Comment:
It's probably not consistent in the existing code, but I prefer `var logCapture bytes.Buffer; &logCapture`

Reply:
Done, and tried to get some more consistency around that.